### ISSUE TITLE
Figma Design Release

### ISSUE THREAD

**Issue by lucasveil** [equinor/design-system#2483]
Planification of Figma release:

- [ ] Select to Autocomplete + documentation
- [ ] Contained button + documentation
- [ ] Sidebar + documentation
- [ ] Icons list

**Comment by lucasveil** [comment]
Documentation for Autocomplete and contained icon are ready to publish in ZeroHeight.
Text for release:

Hi all!
We are pleased to announce a new release in Figma:
- Contained button: as a request from different team, we have implemented a contained icon that can be used as Call to Action. The component is available in all states and color,
- Autocomplete (previously called Select): in order to be aligned with EDS-dev we have updating the name of the select component to Autocomplete. The autocomplete component offers in addition of the select component a normal text input enhanced by a panel of suggested options. Only the name component have been updated and the design remains the same
- New icons for Energy, Hardware, Map transportation, Social, Technical, UI actions and UI Views category
- Sidebar? @BirteThornquist 
Those update are available thanks to all the feedbacks and suggestions from different teams using our design system. 
As you probably now the design team is working on EDS 2.0 and some exciting news will be published very soon :) 
The EDS Core Team
